socorro/unittest/external/test_crashstorage_base.py

```diff
@@ -206,17 +188,11 @@ def save_raw_crash(self, raw_crash, dumps, crash_id):
             def open_function(*args, **kwargs):
                 return values.pop(0)
 
-            crashstorage = MyCrashStorageTest(
-                config,
-                quit_check_callback=fake_quit_check
-            )
+            crashstorage = MyCrashStorage(config, quit_check_callback=fake_quit_check)
 
-            with mock.patch("__builtin__.open") as open_mock:
+            with mock.patch("socorro.external.crashstorage_base.open") as open_mock:
```

Comment:
Python 3 doesn't have a `__builtins__`. I think there's another way to mock builtins, but instead I'm just abusing the `socorro.external.crashstorage_base` namespace which has the open we need to mock in it.